src/chain.h

```diff
@@ -16,6 +16,7 @@
 
 static const int SPROUT_VALUE_VERSION = 1001400;
 static const int SAPLING_VALUE_VERSION = 1010100;
+static const int CHAIN_HISTORY_ROOT_VERSION = 2010200;
```

Comment:
Reviewers: if you want to locally test the reorg part of the RPC test, change this to:
```suggestion
static const int CHAIN_HISTORY_ROOT_VERSION = 2010100;
```